lsst/ts/laser/csc.py

```diff
@@ -39,25 +39,30 @@ class LaserDetailedState(enum.IntEnum):
     PROPAGATINGSTATE = 6
```

Comment:
I would recommend you use the enumeration values from SAL (e.g. https://github.com/lsst-ts/ts_atmonochromator/blob/419b47e1efd40ae9c3cc2661fef83b72ae0a32f0/python/lsst/ts/monochromator/monochromator_csc.py#L46-L64)